### PR TITLE
fixed refs to removed usermod

### PR DESCRIPTION
**Pull request by ingDIY**
cleaned up old references to "USE_ALT_DISPlAY" that aren't used anymore in the code/source files
fixed readme.md documents and updated instructions
removed referencies to old usermods

### CODE REVIEW

File: wled00/const.h [hunk 1]
```diff
@@ -578,8 +578,14 @@
...
-#define DEFAULT_LED_TYPE TYPE_WS2812_RGB
-#define DEFAULT_LED_COUNT 30
+
+#ifndef DEFAULT_LED_TYPE
+  #define DEFAULT_LED_TYPE TYPE_WS2812_RGB
+#endif
+
+#ifndef DEFAULT_LED_COUNT
+  #define DEFAULT_LED_COUNT 30
+#endif
```
Comment (softhack007) [inline wled00/const.h:589]: aehem .... @ingDIY  you know that this part was just rejected as a PR. 

Please remove unrelated changes to keep the PR scope clean.

File: usermods/usermod_v2_four_line_display_ALT/readme.md [hunk 2]
```diff
@@ -74,6 +62,10 @@ These options are configurable in Config > Usermods
 
 Note: the Four Line Display usermod requires the libraries `U8g2` and `Wire`.
 
+## Compatibility
+
+The original "usermod_v2_auto_save" will not work with the display just yet.
+
```
[the remark this answers is not in the document]
Comment (ingDIY) [reply usermods/usermod_v2_four_line_display_ALT/readme.md:68]: I don't know, it was reported in the readme.md

File: wled00/const.h (hunk 1 above)
Comment (ingDIY) [reply wled00/const.h:589]: yes, but I was working on PR on different branches. I did something wrong managing the branches.
should I pull-request from the beginning?

Comment (ingDIY) [reply wled00/const.h:589]: > yes, please start fresh. It maybe best to fork WLED again for having a fresh repo, then making a brach for your PR that just contains the code changes.
> 
> See also:
> 
> * [How to properly submit a PR](https://github.com/Aircoookie/WLED/wiki/How-to-properly-submit-a-PR)
> * https://github.com/Aircoookie/WLED/blob/0_15/CONTRIBUTING.md

ok, i strarted from scratch and made a new PR
